hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -297,7 +310,12 @@ final case class Param(flags: FldFlags, sym: LocalSymbol & NamedSymbol, sign: Op
   // def showDbg: Str = flags.showDbg + sym.name + ": " + sign.showDbg
   def showDbg: Str = flags.showDbg + sym + sign.fold("")(": " + _.showDbg)
 
-object FldFlags { val empty: FldFlags = FldFlags(false, false, false) }
+object FldFlags { 
+  val empty: FldFlags = FldFlags(false, false, false, false)
+
+  // module parameter / module argument, depending on the context
+  val module: FldFlags = FldFlags(false, false, false, true)
```

Comment:
This should not be useful, please remove it. What you want to do with these things is to add some new flags onto an existing value, as in `flags.copy(mod = true)`.

Reply:
So write `FldFlags.empty.copy(mod = true)` instead? Often there are no existing flags for parameters

Reply:
I think I understand what you mean